app/views/searches/_journey.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="govuk-!-margin-top-5">
+  <h3 class="govuk-heading-m govuk-!-margin-bottom-1">
+    <%= link_to journey.title, journey_path(journey.slug), class: 'govuk-link' %>
+  </h3>
+  <p class="govuk-body-s">
+    <%= searched_text(journey.description, query) %>
+  </p>
+  <% steps = PgSearch::Document.where(searchable: journey.steps).search(analyzed_query).page(1).per(4) %>
```

Comment:
Toto by sa malo asi diat v controlleri a mozno nie takto lebo to odpaluje aj 10x (pre kazdy vysledok). Ale nateraz to mergnem nech uz to hlada normalne.